src/mSupplyMobileApp.js

```diff
@@ -278,6 +285,14 @@ class MSupplyMobileAppContainer extends React.Component {
           onAuthentication={this.onAuthentication}
         />
         {isLoading && this.renderLoadingIndicator()}
+        <ModalContainer
+          isVisible={supplierCreditModalOpen}
+          onClose={closeSupplierCreditModal}
+          title={`Available Credits for ${creditItemName}`}
+          fullScreen
+        >
+          <SupplierCredit />
+        </ModalContainer>
```

Comment:
have put this high up in the tree as it would be quite simple to also add a button to supplier invoices which would open the same modal, and populate the table with the transaction batches in the SI rather than item batches from an item - so putting it here will make that super easy in the future when it's inevitably needed 🤣